utility_network/trace-a-utility-network/src/main/java/module-info.java

```diff
@@ -0,0 +1,9 @@
+module trace.a.utility.network.main {
+  requires com.esri.arcgisruntime;
+  requires javafx.base;
+  requires javafx.controls;
+  requires javafx.fxml;
+  requires javafx.graphics;
+
+  exports com.esri.samples.trace_a_utility_network;
+}
```

Comment:
```suggestion
}

```
Best practice to have a blank line at the end of the file. Please also check the other PR too.